src/main/connector/system/Linux/Linux.yaml

```diff
@@ -228,6 +228,32 @@ monitors:
           system.filesystem.usage{system.filesystem.state=free}: $5
           system.filesystem.utilization{system.filesystem.state=used}: $6
           system.filesystem.utilization{system.filesystem.state=free}: $7
+  system:
+    simple:
+      sources:
+        # Distribution;Version;Kernel
+        osInformation:
+          type: commandLine
+          commandLine: |
+            os_info=$(cat /etc/os-release | grep -E '^(NAME|VERSION="[0-9].*")' | tr -d '"')

```

Comment:
It's actually more efficient and robust to "execute" `/etc/os-release` and then echo the required env variables. The below script will achieve the same result with much less processes:

```bash
. /etc/os-release
echo "$NAME;$VERSION;`uname -r`"
```